packages/elements/src/tree/elements/tree.ts

```diff
@@ -37,9 +37,10 @@ export class Tree<T extends TreeDataItem = TreeDataItem> extends List<T> {
   protected override readonly defaultRole: string | null = 'tree';
 
   /**
-   * Tree manager used for manipulation
+   * Tree manager used for item manipulation
+   * @type {TreeManager<T>}
    */
-  private manager = new TreeManager<T>(this.composer);
+  public manager: TreeManager<T> = new TreeManager<T>(this.composer);
```

Comment:
It must be encapsulated and called by getter public API to prevent a developer set/unset the `manager` that might break the component.

Reply:
adopted your suggestion. 👍🏼